### PR TITLE
Module 6 add architecture diagrams

### PR DESCRIPTION
**Pull request by 11braemm**
Please make sure you are raising this PR against your own repository and not the original. If it says https://github.com/CorndelWithSoftwire in the search bar right now, you are in the wrong place!

Please also don't forget to submit the PR's url to Aptem afterwards (by clicking on the corresponding exercise component, pasting the url in the textbox that appears and then pressing the finish button).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,8 +8,10 @@ The project uses poetry for Python to create an isolated environment and manage
 
 ### Poetry installation (Bash)
 
+Note you will need to install Python for this.
```
Comment (11braemm) [inline README.md:11]: I had to set this up on a new laptop so added some steps to the README where I had issues

File: todo_app/data/session_items.py
Comment (11braemm) [inline todo_app/data/session_items.py:1]: deleted this as no longer using

File: todo_app/test_view_model.py
Comment (11braemm) [inline todo_app/test_view_model.py:1]: moved this as a test file should not be in /data

**Review by JackMeadDev — APPROVED** [review]
Nice work, you've got clean simple diagrams that clearly show different levels of detail intended for different audiences! I've left some more detailed thoughts below for consideration, but I'd advise just going ahead and merging as they are! 😄 

Context:
* Nice - really clear and simple
* You've used a few different colours, and it's not really clear to me if they're meant to mean anything - if they are, then potentially a key would be useful!
  * And possibly covered by that, but one addition I like here is to clearly label Trello as external; while that may be obvious to anyone who's used it, it's easy to imagine a stakeholder misunderstanding and thinking we have more control over that than we do (e.g. we can't easily control what country those servers store data in, or how backups run)

Container:
* Nice - simple & clear again, and particularly the addition of the browser makes it really clear how client-side code is interacting with our container!

Component:
* Again, really clear and ties in easily with the previous diagram
* Good level of detail - gives the structure of the code without worrying about unnecessary detail

**Comment by 11braemm** [comment]
The colours were just for style rather than meaning so no need for a key, I'll add labelling Trello as external and merge it :)

**Pull request merged**